Update DID Web Discovery tutorial (#197)
w3c-ccg/traceability-interop: docs/tutorials/did-web-discovery/README.md

# DID Web Discovery Tutorial

If you have not done so already, please review and complete the previous tutorials which form the foundation for the material presented in this tutorial.

1. [Access Token Tutorial](../authentication/README.md) - Obtain an OAuth2 client credentials grant token

In this tutorial, we will be using DID Web Discovery to find the base URL for a VC-API that supports presentation exchange as well as a DID to use for issuing credentials.

## Setup and Configuration

> This tutorial will be picking up where the Access Token Tutorial left off; the quickest way to get set up to work on this tutorial is to:

Reviewer: ```suggestion
This tutorial picks up where the Access Token Tutorial left off; the quickest way to get set up to work on this tutorial is to:
```

> 1. Duplicate the Access Token Tutorial and rename it to DID Web Discovery Tutorial.
> 1. Duplicate the Access Token Environment and rename it to DID Web Discovery Environment.

Reviewer: ```suggestion
1. Duplicate the `Access Token Tutorial` and rename it to `DID Web Discovery Tutorial`.
1. Duplicate the `Access Token Environment` and rename it to `DID Web Discovery Environment`.
```

Be sure to activate the new DID Web Discovery Environment by selecting it in the drop-down menu at the top-right of the Postman interface.

<img src="./resources/select-environment.png"/>

## Access Token Request

> The first thing that we need to do is preserve the output of the Get Access Token request so that we can use the created access token for authenticated requests to the VC API endpoint.

Reviewer: ```suggestion
First, preserve the output of the `Get Access Token` request, so that we can use the created access token for authenticated requests to the VC API endpoint.
```

Open up the "Get Access Token" request in the "Get Access Token Tutorial" and select the "Tests" tab. Add the following code to the end of the test so that the access token is stored as a collection variable:

```javascript
// The returned `access_token` value is persisted as a Postman collection
// variable that can be accessed by other requests in the collection by calling
// `pm.collectionVariables.get("access_token")`.
pm.test("`access_token` persisted to collectionVariables", function() {
    const { access_token } = pm.response.json()
    pm.collectionVariables.set("access_token", access_token);
});
```

When you are done, the "Tests" tab should look something like this:

<img src="./resources/persist-access-token.png"/>

You will need to run this request at least once in order to populate the `access_token` collection variable before you can proceed with the next request.

## Get Organization DIDs

> In this tutorial, we will be using DID Web Discovery to find the base URL for a VC-API that supports presentation exchange as well as a DID to use for issuing credentials.

Reviewer: ```suggestion
In this tutorial, we use DID Web Discovery to find the base URL for a VC-API that supports presentation exchange as well as a DID to use for issuing credentials.
```

### Environment

For this tutorial, you will need to add the following environment variables to the DID Web Discovery Environment:

<dl>
  <dt><code>API_BASE_URL</code></dt>
  <dd>The base URL for your organizations VC-API supporting DID resolution.</dd>
  <dt><code>ORGANIZATION_DID_WEB</code></dt>

>   <dd>Your organization's DID Web, <em>e.g.</em>, <code>did:web:api.did.actor:api</code></dd>

Reviewer: ```suggestion
  <dd>Your organization's DID Web, e.g., <code>did:web:api.did.actor:api</code></dd>
```
